examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pageheader/.content.xml

```diff
@@ -1,129 +1,122 @@
-                        jcr:primaryType="nt:unstructured"
-                        jcr:title="Standard"
-                        sling:resourceType="core/wcm/components/title/v2/title"
-                        type="h3"/>
+                    cq:styleIds="[1544759676459]"
+                    jcr:lastModified="{Date}2019-05-22T11:01:40.926+03:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Standard"
+                    sling:resourceType="core/wcm/components/title/v2/title"
+                    type="h3"/>
                 <text_1337506761
-                        jcr:created="{Date}2018-12-06T19:23:41.968+01:00"
-                        jcr:createdBy="admin"
-                        jcr:lastModified="{Date}2019-05-22T11:26:32.604+03:00"
-                        jcr:lastModifiedBy="admin"
-                        jcr:primaryType="nt:unstructured"
-                        sling:resourceType="core/wcm/components/text/v2/text"
-                        text="&lt;p>The component, simply displaying placeholder data.&lt;/p>"
-                        textIsRich="true"/>
+                    jcr:lastModified="{Date}2019-05-22T11:26:32.604+03:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="core/wcm/components/text/v2/text"
+                    text="&lt;p>The component, simply displaying placeholder data.&lt;/p>"
+                    textIsRich="true"/>
                 <demo
-                        jcr:created="{Date}2018-12-07T12:55:03.496+01:00"
-                        jcr:createdBy="admin"
-                        jcr:lastModified="{Date}2018-12-07T12:55:03.496+01:00"
-                        jcr:lastModifiedBy="admin"
-                        jcr:primaryType="nt:unstructured"
-                        sling:resourceType="forms-components-examples/components/demo">
+                    jcr:lastModified="{Date}2018-12-07T12:55:03.496+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="forms-components-examples/components/demo">
                     <component
-                            jcr:primaryType="nt:unstructured"
-                            sling:resourceType="core-components-examples/components/demo/component">
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="core-components-examples/components/demo/component">
                         <container
-                                jcr:primaryType="nt:unstructured"
-                                sling:resourceType="core-components-examples/components/form/container">
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="core-components-examples/components/form/container">
```

Comment:
FYI : @amit-mnnit is fixing this part where RT of form should point to our component. without that, our runtime clientlibs are not loaded.